test/e2e_node/container_manager_test.go

```diff
@@ -100,7 +100,32 @@ var _ = SIGDescribe("Container Manager Misc", framework.WithSerial(), func() {
 					return validateOOMScoreAdjSetting(kubeletPids[0], -999)
 				}, 5*time.Minute, 30*time.Second).Should(gomega.BeNil())
 			})
-			ginkgo.Context("", func() {
+
+			ginkgo.Context("with test pods", func() {
+				var testPod *v1.Pod
+
+				// Log the running containers here to help debugging.
+				ginkgo.AfterEach(func(ctx context.Context) {
+					if ginkgo.CurrentSpecReport().Failed() {
+						ginkgo.By("Dump all running containers")
+						runtime, _, err := getCRIClient()
+						framework.ExpectNoError(err)
+						containers, err := runtime.ListContainers(ctx, &runtimeapi.ContainerFilter{
+							State: &runtimeapi.ContainerStateValue{
+								State: runtimeapi.ContainerState_CONTAINER_RUNNING,
+							},
+						})
+						framework.ExpectNoError(err)
+						framework.Logf("Running containers:")
+						for _, c := range containers {
+							framework.Logf("%+v", c)
+						}
+					}
+
+					deletePodSyncByName(ctx, f, testPod.Name)
```

Comment:
what if test pod was not assgned yet? Can we add a check for this please

Reply:
Right, thanks. Fixed.